install/froxlor.sql

```diff
@@ -709,7 +709,7 @@ opcache.interned_strings_buffer'),
 	('panel', 'customer_hide_options', ''),
 	('panel', 'is_configured', '0'),
 	('panel', 'version', '0.10.24'),
-	('panel', 'db_version', '202101200');
+	('panel', 'db_version', '202102200');
```

Comment:
You also need to add the new setting here for new installations:

Reply:
silly me…fixed